src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -975,6 +975,29 @@ public class ShowHideConsoleMenuItemConverter : IValueConverter
         #endregion
     }
 
+    /// <summary>
+    /// Obsolete class of ShowHidePreviewBubblesConverter
+    /// Using of this class will produce compile warnings
+    /// TODO: To be removed in Dynamo 2.0
+    /// </summary>
+    [System.Obsolete("This class is obsolete, string converter for Preview Bubble Display Option only returns single value now. ")]
+    public class ShowHidePreviewBubblesConverter : IValueConverter
+    {
+        #region IValueConverter Members
+
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return Resources.DynamoViewSettingsMenuShowPreviewBubbles;
```

Comment:
It will not cause crash, the call in 1.2 will only return single value "show xxx" instead of "show xxx" or "hide xxx" in 1.0 and 1.1.